sonar-project.properties

```diff
@@ -6,7 +6,7 @@ sonar.tests=src/test/
 sonar.test.exclusions=src/**/*.njk
 sonar.typescript.lcov.reportPaths=coverage/lcov.info
 sonar.javascript.lcov.reportPaths=coverage/lcov.info
-sonar.exclusions=src/**/*.test.ts,**/__mocks__/**,src/**/*content.ts,src/**/components/*.ts,src/**/definition.ts,src/**/summary/util.ts,src/**/summary/util.ts,src/**/steps/constants.ts,src/**/willingness-to-attend-miam/miam-cost-exemptions.ts,src/**/steps/index.ts,src/**/steps/urls.ts,src/**/case/CaseApi.ts,src/**/app/controller/PostController.ts,src/**/checkanswers/ConfirmContactDetailsGetController.ts,src/**/respondent/task-list/utils.ts,src/**/respondent/respondentcaseSequence.ts,src/main/steps/common/summary/utils.ts
+sonar.exclusions=src/**/*.test.ts,**/__mocks__/**,src/**/*content.ts,src/**/components/*.ts,src/**/definition.ts,src/**/summary/util.ts,src/**/summary/util.ts,src/**/steps/constants.ts,src/**/willingness-to-attend-miam/miam-cost-exemptions.ts,src/**/steps/index.ts,src/**/steps/urls.ts,src/**/case/CaseApi.ts,src/**/app/controller/PostController.ts,src/**/checkanswers/ConfirmContactDetailsGetController.ts,src/**/respondent/task-list/utils.ts,src/**/respondent/respondentcaseSequence.ts,src/main/steps/common/summary/utils.ts,src/**/checkanswers/RespondentConfirmContactDetailsGetController.ts,src/**/checkanswers/ApplicantConfirmContactDetailsGetController.ts
```

Comment:
why we need to add these controllers to sonar exclusions?

Reply:
removed from the exclusions, implemented the unit tests for the same.